apps.yml

```diff
@@ -602,18 +556,6 @@ apps:
     url: https://developer.mozilla.org/
     vanity_url:
     - /mdn
-- application:
-    authorized_groups:
-    - everyone
-    authorized_users: []
-    client_id: nPr8QRo0dLxM3RRHwIXRSvhmOSPDvNr4
-    display: true
-    logo: moderator.png
-    name: Moderator
-    op: auth0
-    url: https://moderator.mozilla.org/oidc/authenticate/
-    vanity_url:
-    - /moderator
```

Comment:
sso.m.c/moderator still responds - there's a 'dupe' entry down at `cav8o4za5QGMXilUEjglH9cgJpQl33Ck`.  Does that other entry need to pick up the vanity url?